src/Trixi.jl

```diff
@@ -170,16 +169,12 @@ export flux, flux_central, flux_lax_friedrichs, flux_hll, flux_hllc, flux_hlle,
        flux_kennedy_gruber, flux_shima_etal, flux_ec,
        flux_fjordholm_etal, flux_nonconservative_fjordholm_etal,
        flux_wintermeyer_etal, flux_nonconservative_wintermeyer_etal,
-       flux_es_ersing_etal, flux_nonconservative_ersing_etal,
+       flux_nonconservative_ersing_etal,
```

Comment:
`flux_nonconservative_ersing_etal` is just for standard shallow water?

Reply:
Yes, it's basically just a reformulation of `flux_nonconservative_wintermeyer_etal`, such that it can also be used at the surface. It is not related to the wet/dry treatment and can therefore stay in `Trixi.jl`.